server/src/main/java/io/crate/planner/Planner.java

```diff
@@ -435,6 +435,9 @@ public Plan visitRestoreSnapshotAnalyzedStatement(AnalyzedRestoreSnapshot analys
     @Override
     public Plan visitSetStatement(AnalyzedSetStatement setStatement, PlannerContext context) {
         switch (setStatement.scope()) {
+            case TIME_ZONE:
+                LOGGER.info("SET TIME ZONE `{}` statement will be ignored. ", setStatement.settings());
```

Comment:
```suggestion
                LOGGER.warn("SET TIME ZONE `{}` statement will be ignored. ", setStatement.settings());
```

Given that the time zone is not actually used and that users could expect some operations to behave differently depending on the time zone I think `warn` is warranted here.

Reply:
I think the problematic thing, still is, that this will spam the CrateDB logs with lots of entries, when using "standard" postgres drivers / clients, even when "setting" it to `UTC`

Reply:
We could add an exception for `UTC` to avoid logging in that case. In any other case the behavior is strictly speaking incorrect and just swallowing that is imho worse than log spam (which should only happen once per connection setup, if that happens all the time, it is a sign of clients being too aggressive with connection opening/closing anyways)

Reply:
Should probably be a separate issue, as this should be reflected in both the generic `SET` and `SET TIME ZONE`.

> if that happens all the time, it is a sign of clients being too aggressive with connection opening/closing anyways

Can't influence what others are doing 😂

Reply:
ok, so should we skip the logging when the time zone is `UTC` ?